src/docfx/lib/log/Log.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class Log
+    {
+        private static AsyncLocal<bool> t_verbose = new AsyncLocal<bool>();
+
+        public static IDisposable BeginScope(bool verbose)
+        {
+            t_verbose.Value = verbose;
+            return new LogScope(() => t_verbose.Value = false);
+        }
+
+        public static void Info(string message)
```

Comment:
`Info` is not in use for now? Maybe it can be consolidated with `Error(string message)`, sharing most logic